### ISSUE TITLE
bind irx4 lite with multiplex m-link receiver

### ISSUE THREAD

**Issue by Franz-Royer** [FrSkyRC/ETHOS-Feedback-Community#4883]
I can't bind an rx7dr light m_link receiver with the irx4 lite multi.
Firmware ethos 1.5.19 EU on frsky x20.
Firmware irx4 lite is 1.3.4.0
Luascript is installed on the transmitter, latest version.
Can anyone help me?

Lg Franz

**Comment by mawzthefinn** [comment]
Have you asked the MPM folks? That sounds more like an MPM issue. M-link support on MPM is pretty limited as far as I;m aware.

**Comment by bsongis-frsky** [comment]
Indeed, not sure it is related to Ethos!

**Comment by Franz-Royer** [comment]
The module works, I can bind the Spektrum receiver. I can also bind the Goosky s2 without any problems.

I have already reset the M-Link receiver.

**Comment by Franz-Royer** [comment]
no matter how I try it always flashes the same

**Comment by mawzthefinn** [comment]
Yep, you need to take this to Pascal at the MPM github. This isn't an ETHOS issue, it's an MPM issue.

**Comment by mawzthefinn** [comment]
Closing as this is an MPM issue, not an ETHOS issue
